### DISCUSSION TITLE
FsLeyes won't finish creaing the interface

### DISCUSSION THREAD

**Discussion by EdnaSP** [neuropoly/axondeepseg#725] [Help needed] [answered]
Good morning, I am using windows and trying to install Axondeepseg since yesterday, I followed the instructions, and axondeepseg was installed successfully, but when things changed when trying to use FSLeyes, I waited for 24hrs for the interface to be created and nothing is happening, can you please help? Note: I am new at this coding thing. 

Thank you in advance

**Comment by hermancollin** [comment, marked as answer]
Hi @EdnaSP. Thank you for reaching out. Could you provide a bit more context? When you say you tried using FSLeyes, do you mean you typed `fsleyes` in the terminal and waited for it to load for 24 hours? Or did it successfully open and you tried loading the axondeepseg plugin but it failed?

**Reply by mathieuboudreau** [reply]
@hermancollin it sounds like this user is trying to run FSLeyes in Windows, 

 > I am using windows and trying to install Axondeepseg since yesterday

@EdnaSP FSLeyes doesn't support Windows (this is mentioned in [our documentation](https://axondeepseg.readthedocs.io/en/latest/documentation.html?highlight=fsleyes#install-axondeepseg)). Instead, we've been developing an alternative plugin using the software Napari. We're still testing it, but feel free to try it in the current beta stage: https://github.com/axondeepseg/napari-plugin

**Reply by EdnaSP** [reply]
yes I am trying to windows, I will give the Napari a try then thank you

**Reply by EdnaSP** [reply]
I installed napari successfully, thank you. I will give it a try and I'll get back to you if I run into trouble thank you so much.

**Comment by EdnaSP** [comment]
Hi, thank you for getting back to me, I yes I did typed fsleyes in the terminal and a separated window opened,  started loading, then changed to creating a FLSeyes interface... and nothing happed after, I tryed even using an older python version as suggested in other discussion but still the same result
